.eslintrc.js

```diff
@@ -239,10 +239,7 @@ module.exports = {
         ],
         "vue/max-attributes-per-line": ["error", {
             "singleline": 1,
-            "multiline": {
```

Comment:
@trusktr , this was done on purpose because of styling preferences of the team. FWIW, I would prefer multiline 1, period. I would check with the team first before we push this change.

Reply:
cc @akhenry , @nikhilmandlik , @shefalijoshi , @charlesh88, @jvigliotta

Reply:
@davetsay `multiline: 1` is equivalent to `multiline: {max: 1}`, and the latest docs don't mention any `allowFirstLine` option so this is as close as the option can remain if we update eslint-vue.